config/ide-helper.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+return array(
```

Comment:
tl;dr assuming you are happy with these defaults.

Reply:
Yep

Reply:
Ace!